packages/govuk-frontend/src/govuk/overrides/_typography.scss

```
@@ -3,8 +3,13 @@
 
   // Generate typography override classes for each responsive font map in the
   // typography scale eg .govuk-\!-font-size-80
+  //
+  // govuk-!-font-size-14 is deprecated
   @each $size in map-keys($govuk-typography-scale) {
     .govuk-\!-font-size-#{$size} {
+      @if $size == 14 {
+        $size: _14;
+      }
```

Do we need to update the `govuk-font-size()` SassDoc type to allow strings?

```scss
/// @param {String | Number} $size - Point from the spacing scale (the size as
///   it would appear on tablet and above)
```

---

We should probably check for the `"deprecation"` key here too?

Other sizes could be deprecated in future so should avoid hard coding them

```suggestion
  @each $size, $font-map in $govuk-typography-scale {
    $font-map: map-get($govuk-typography-scale, $size);

    // Add underscore to deprecated typography scale keys
    @if map-has-key($font-map, "deprecation") {
      $size: _#{$size};
    }
```

---

Going to unresolve this one based on https://github.com/alphagov/govuk-frontend/pull/4649#pullrequestreview-1856265873

Both approaches renamed `14` to `_14`, is that where the extra class `.govuk-!-font-size-_14` comes from?

Or because we're looping both `14` and `_14` in `$govuk-typography-scale`?

---

It's the latter. Just checking a solution now.

---

I'm going to re-resolve this! CSS diff is now zero.